### PR TITLE
Fix description in package.json

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
@valdrinkoshi, @azakus, please merge this pull request.

**Comment by notwaldorf** [comment]
Needs a rebase

**Comment by FluorescentHallucinogen** [comment]
@notwaldorf
> Needs a rebase

Done.

**Comment by FluorescentHallucinogen** [comment]
@valdrinkoshi, please merge this pull request.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "webcomponents.js",
   "version": "1.0.0-rc.8",
-  "description": "webcomponents.js",
+  "description": "Web components polyfill",
```
Comment (abdonrd) [inline package.json]: Perhaps this description would be better?

>A suite of polyfills supporting the Web Components specifications

Comment (FluorescentHallucinogen) [inline package.json]: 9 words vs 3. )))

Comment (abdonrd) [inline package.json]: Yes, but it is a description. haha :)

Comment (FluorescentHallucinogen) [inline package.json]: Do polyfills `supports` specifications? Polyfills _implement_ features (specification support) that not supported by browsers natively.

WebComponents.js is more than just a suite, it's a dynamic loader uses feature detection, ES5 adapter, etc...

Anyway, it's much better than original description — `webcomponents.js`.

Feel free to change the description after merge this pull request. But in this case don't forget to change it in `bower.json` file too.

**Comment by FluorescentHallucinogen** [comment]
I believe we should use the same description in both `bower.json` and `package.json` files.

**Comment by TimvdLippe** [comment]
Since we no longer have `bower.json`, I am closing this PR.

**Pull request closed without merging**